### ISSUE TITLE
[Feature Request] Option to move player controls below video (new html5 player)

### ISSUE THREAD

**Issue by Oddish** [YePpHa/YouTubeCenter#1991]
I'm very annoyed at the new html5 player which puts the player controls on top of the video and auto shows them on hover. No longer can I see the length of the video and how much is left of it, without hovering the mouse over the video. I don't know if it's possible but I would LOVE and option in YTC that would move the controls below the video and get rid of the auto hide/show.

**Comment by Yonezpt** [comment]
It is possible, however the time and progress bar will "freeze" whenever the mouse is not over the video. This is per their design.

**Comment by ackern** [comment]
I agree the new design is a catastrophe.. There's really no way to get the old style back?

**Comment by Yonezpt** [comment]
For now the only way of getting the old style back is by using the flash player, it is still using the previous layout.

**Comment by Piter432** [comment]
@ParticleCore Nope. We can move player controls below video on new HTML5
player, because of this >
https://greasyfork.org/pl/scripts/11485-youtube-new-ui-fix < userscript.

**Comment by Yonezpt** [comment]
@Piter432 I never said it was not possible. The issue with the solution in the script you linked is that it will impact performance, especially noticeable for people that have slower computers. It is not a good solution and will result in people complaining about how the script is eating their resources.

I leave this up to YePpHa, maybe he knows of a better way.
